### DISCUSSION TITLE
PhpOffice\PhpSpreadsheet\Collection\CacheInterface not found

### DISCUSSION THREAD

**Discussion by gdanen** [PHPOffice/PhpSpreadsheet#2513] [General]
I have installed PhPSpreadsheet using Composer and PhpOffice\PhpSpreadsheet\Collection\CacheInterface seems to be missing.
What to do?

TIA
Gerry

**Comment by gdanen** [comment]
I was trying to manually include dependencies, as I am not able to run composer on my shared host server.
It was suggested to install locally and then let composer bring in the files, after which the relevant files are uploaded to the server.

**Comment by MarkBaker** [comment]
It's not `PhpOffice\PhpSpreadsheet\Collection\CacheInterface`, it's `Psr\SimpleCache\CacheInterface`, which is included as part of the `psr/simple-cache` dependency. It should be in the `vendor/psr/simple-cache/src/` folder. and made available within the autoloader when you include `vendor/autoload.php`.
